<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keywidget</class>
 <widget class="QWidget" name="keywidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>218</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>OPPOSans B</family>
    <pointsize>10</pointsize>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>导入私钥</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lib.qrc">
    <normaloff>:/lib/wicon_46.png</normaloff>:/lib/wicon_46.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 45, 54);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QToolButton" name="toolButton_ok">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>OPPOSans B</family>
     <pointsize>10</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 95, 95);
	border-radius: 5px;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 95, 95);
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_cancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>OPPOSans B</family>
     <pointsize>10</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 95, 95);
	border-radius: 5px;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 95, 95);
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_key_pssword">
      <property name="font">
       <font>
        <family>OPPOSans B</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>私钥密码：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_key">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 25, 25, 155);
border: 1px solid rgba(255, 255, 255, 0);
border-radius: 5px;
padding-left:10px;
padding-right:10px;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_key_pssword">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 25, 25, 155);
border: 1px solid rgba(255, 255, 255, 0);
border-radius: 5px;
padding-left:10px;
padding-right:10px;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_name">
      <property name="font">
       <font>
        <family>OPPOSans B</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>名称：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_key">
      <property name="font">
       <font>
        <family>OPPOSans B</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>私钥：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(25, 25, 25, 155);
border: 1px solid rgba(255, 255, 255, 0);
border-radius: 5px;
padding-left:10px;
padding-right:10px;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QToolButton" name="toolButton_browse">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>OPPOSans B</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 95, 95);
	border-radius: 5px;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 95, 95);
}</string>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="lib.qrc"/>
 </resources>
 <connections/>
</ui>
